src/fsipy/automatedPostprocessing/postprocessing_fenics/postprocessing_fenics_common.py

```diff
@@ -24,3 +26,29 @@ def parse_arguments() -> argparse.Namespace:
                         help="Specify the log level (default is 20, which is INFO)")
 
     return parser.parse_args()
+
+
+def project_dg(f, V) -> Function:
+    """
+    Project a function v into a DG space V.
+    It perfomrs the same operation as dolfin.project, but it is more efficient
+    since we use local_solver which is possible since we use DG spaces.
+
+    Args:
+        v (dolfin.Function): Function to be projected
+        V (dolfin.FunctionSpace): DG space
+        dx_s (dolfin.Measure): Measure over the solid domain
+
+    Returns:
+        dolfin.Function: Projected function
+    """
+    v = TestFunction(V)
+    u = TrialFunction(V)
+    a = inner(u, v) * dx
+    L = inner(f, v) * dx
+    u_ = Function(V)
+    solver = LocalSolver(a, L)
+    solver.factorize()
+    solver.solve_local_rhs(u_)
```

Comment:
We might nee to use `solve_global_rhs` here, if the input `f` is not in `DG`.
We should probably also add a check to ensure that `V` is a DG space.